content/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -5,53 +5,139 @@ weight = 50
 +++
 
 This page describes authentication for Kubeflow Pipelines to GCP.
+Available options listed below have different tradeoffs. You should choose the one that fits your use-case.
+
+* [Compute engine default service account](#compute-engine-default-service-account) is easy to set up, but overgrants permission. Therefore, it is not suitable for a shared GCP project.
```

Comment:
but overgrants permission if enabled access to "cloud-platform" scope.

Reply:
good point